### PR TITLE
Content/commiters fix

### PR DESCRIPTION
**Pull request by kgabryje**

### CODE REVIEW

File: landing-pages/site/assets/scss/_list-boxes.scss [hunk 1]
```diff
@@ -63,9 +63,11 @@ $card-margin: 20px;
   border-radius: 5px;
   margin: $card-margin;
   padding: 30px 10px;
+  height: calc(100% - #{$card-margin} * 2);
```
Comment (mik-laj) [inline landing-pages/site/assets/scss/_list-boxes.scss]: Can you do it without calc? For example:: `align-items: stretch`

File: landing-pages/site/assets/scss/_list-boxes.scss [hunk 2]
```diff
@@ -39,11 +40,12 @@ $card-margin: 20px;
 
 .list-item {
   width: 25%;
-
+  padding: $card-margin;
   @media (min-width: $fullhd) {
     width: 20%;
   }
   @media (max-width: $tablet) {
+    padding: 1.5 * $card-margin;
```
Comment (kgabryje) [inline landing-pages/site/assets/scss/_list-boxes.scss]: Why *1.5? It was 0.5 before

File: landing-pages/site/assets/scss/_list-boxes.scss (hunk 1 above)
Comment (kgabryje) [reply landing-pages/site/assets/scss/_list-boxes.scss]: resolved

File: landing-pages/site/assets/scss/_list-boxes.scss (hunk 2 above)
Comment (kgabryje) [reply landing-pages/site/assets/scss/_list-boxes.scss]: updated

**Pull request merged**